native/helpers/challenges.js

```diff
@@ -31,6 +31,14 @@ export const challenges = [
     youtube: 'https://youtu.be/crCZEJ6Lmgc',
     tags: [],
   },
+  {
+    title: 'FileUploader',
+    link: 'fileuploader',
```

Comment:
link should be in lowercase (this is fine) but with a hyphen to separate the words, i.e `file-uploader`